src/EnergyPlus/DataHeatBalSurface.hh

```diff
@@ -129,6 +129,9 @@ namespace DataHeatBalSurface {
     extern Array1D<Real64> QRadOutReport;        // Surface thermal radiation heat gain at Outside face [J]
     extern Array1D<Real64> QdotRadOutRep;        // Surface thermal radiation heat transfer outside face surface [W]
     extern Array1D<Real64> QdotRadOutRepPerArea; // [W/m2]Surface thermal radiation heat transfer rate per m2 at
+
+    extern Array1D<Real64> QAirExtReport;  // Surface Outside Face Thermal Radiation to Air Heat Transfer Rate [W]
```

Comment:
Seems that Surface Outside Face Thermal Radiation to *Ground* Heat Transfer Rate should also be tracked and added to `QHeatEmiReport`.
In the later calculations of this, I was confused by variable `HAirExtSurf` and noticed that [these old comments are wrong](https://github.com/NREL/EnergyPlus/blob/e9cf73860342b899068a32749a1cff5ff50aa9cb/src/EnergyPlus/DataHeatBalSurface.cc#L98-L100). Please fix them.

*End MJW review here for today.*

Reply:
@mjwitte I'm not sure about this. As in the NFP this feature is to calculate the heat emission to ambient **air**, so Radiation to Ground or Sky may not be considered.

Reply:
I agree that radiation to sky would not be counted, but some portion of radiation to ground will ultimately end up in the air. I haven't reviewed the docs yet for this, but it should be made clear that radiant exhange to the sky and ground are not counted.

Reply:
Added clarifications in the docs.